docs/amphtml/register-outlook.md

```diff
@@ -8,22 +8,25 @@ ms.author: humlezg
 localization_priority: Priority
 ---
 
-# Sender Registration
+# Sender registration
+
 The email address AMP emails originate from must be registered with Outlook to render AMP emails. If the address is not registered the AMP part will be ignored and the HTML version of the email will be rendered instead. There are two mechanisms to register an address with Outlook.
 
-## Individual Mailbox registration
+## Individual mailbox registration
+
 During development, you can self-register to test AMP emails on your account. Follow these steps:
 
-1. [Login](https://www.outlook.com) to you outlook.com account
-1. Go to `Settings>All Outlook settings>Dynamic Email`
-1. Ensure that dynamic email is turned on
-1. Add any sender email addresses that you will use during your testing
+1. [Login](https://www.outlook.com) to you Outlook.com account.
+1. Select **Settings**, then **View all Outlook settings**, then **Dynamic Email**.
+1. Ensure that dynamic email is enabled.
+1. Add any sender email addresses that you will use during your testing.
 
 [add screenshot showing how settings look like]
 
-## Global Registration
-Once you are ready to send AMP emails to Outlook.com users (your customers) then you must register and be approved by Microsoft. Follow these steps:
+## Global registration
+
+Once you are ready to send AMP emails to Outlook.com users (your customers), then you must register and be approved by Microsoft. Follow these steps:
 
-1. Ensure you meet all [security requirements](security-requirements)
-1. [Fill out this form](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbRzX-CbfWK8dJr5uYgzqdeDJUMkRSRFpJUEoxRUVOTFpXVEpWR0xJVlpSTy4u) to request becoming an authorized sender 
-1. We will get back to you with a decision
\ No newline at end of file
+1. Ensure you meet all [security requirements](security-requirements.md).
+1. [Fill out this form](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbRzX-CbfWK8dJr5uYgzqdeDJUMkRSRFpJUEoxRUVOTFpXVEpWR0xJVlpSTy4u) to request registration as an authorized sender.
+1. Our team will review your request and contact you with a decision.
```

Comment:
From experience with the actionable message registration process, this kind of thing generates a lot of questions. Mostly because they don't know what to expect: a response in hours, or weeks? I know it's hard to pin this kind of thing down, but if we could at least set some sort of expectation here it would help.

Reply:
Good feedback. I'll chat with the FC to see if we can commit to a time frame.